### ISSUE TITLE
Support PDF/UA-1

### ISSUE THREAD

**Issue by MBradbury** [jomaway/typst-bytefield#50]
When compiling with `--pdf-standard ua-1` I get the following error message:

```
error: PDF/UA-1 error: missing alt text
    ┌─ @preview/bytefield:0.0.8/lib/api.typ:130:30
    │
130 │     _second = layout(size => {$ stretch(\{,size:size.height) $})
    │                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    │
    = hint: make sure your images and equations have alt text
```

This looks similar to https://github.com/Dherse/codly/pull/114 and https://github.com/swaits-typst-packages/codly-languages/pull/4, so wrapping the equation in `pdf.artifact` should fix the error. To me this does not seem an equation that needs an alt text.
